### DISCUSSION TITLE
Error in Tutorial: Integrating scRNA-seq and scATAC-seq data

### DISCUSSION THREAD

**Discussion by jahanzebsaqib** [satijalab/seurat#5207] [Q&A]
Hope you are doing well!

I am a beginner in the bioinformation and especially in the field of scRNA seq analysis. I was following the Seurat tutorials when I got to the above tutorial I faced the following error, previously it was working perfectly but from the past few days it's not working I have already updated all the packages but still, the problem is persisting. Kindly you are requested to review the matter, your kind consideration will be highly regarded by all means.

Link of tutorial: [Integrating scRNA-seq and scATAC-seq data](https://satijalab.org/seurat/articles/atacseq_integration_vignette.html).

# load both modalities
```
pbmc.rna <- LoadData("pbmcMultiome", "pbmc.rna")
pbmc.atac <- LoadData("pbmcMultiome", "pbmc.atac")
```

# repeat QC steps performed in the WNN vignette
```
pbmc.rna <- subset(pbmc.rna, seurat_annotations != "filtered")
pbmc.atac <- subset(pbmc.atac, seurat_annotations != "filtered")
```

# Perform standard analysis of each modality independently RNA analysis
```
pbmc.rna <- NormalizeData(pbmc.rna)
pbmc.rna <- FindVariableFeatures(pbmc.rna)
pbmc.rna <- ScaleData(pbmc.rna)
pbmc.rna <- RunPCA(pbmc.rna)
pbmc.rna <- RunUMAP(pbmc.rna, dims = 1:30)
```

# ATAC analysis add gene annotation information
`annotations <- GetGRangesFromEnsDb(ensdb = EnsDb.Hsapiens.v86)
``
**The code is working perfectly till this part but when I tried to run the next line**

`seqlevelsStyle(annotations) <- "UCSC"
`

**it gives the following error**

Error in .order_seqlevels(chrom_sizes[, "chrom"]) : 
  !anyNA(m32) is not TRUE

**Comment by mzz92** [comment]
try:

library(EnsDb.Hsapiens.v86)
library(BSgenome.Hsapiens.UCSC.hg38)

annotations <- GetGRangesFromEnsDb(ensdb = EnsDb.Hsapiens.v86)
seqlevelsStyle(annotation) = "UCSC"
genome(annotation) = "hg38"

**Comment by samuel-marsh** [comment]
Hi,

I don't get this error.  Could you paste your sessionInfo here.

Best,
Sam
